Client/Frontend/Menu/Menu.xcassets/menu-ForgotMode.imageset/Contents.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Don't we have this asset already? Would be beter to use same icon

Reply:
Done